### PR TITLE
Dev/josemesona/simplify

### PR DESCRIPTION
**Pull request by josemesona**
#2 Simplifying Core properties.

1. Replaced some properties with constants:
  - CBTModuleConfigPath
 - CBTModulePropertyNamePrefix
 - CBTModulePropertyValuePrefix
 - CBTModuleImportRelativePaths
2. Moved some properties from before.build.props to root build.props so it could be deleted.
3. Removed CBTLocalBuildExtensionsPath that contained the MSBuild version

### CODE REVIEW

File: build.props [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <EnlistmentRoot>$(MSBuildThisFileDirectory)</EnlistmentRoot>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)\obj</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)obj</BaseIntermediateOutputPath>
```
Comment (kingerja) [inline build.props]: Due to context my point here isn't an issue.
However if this enlistmentroot was being set elsewhere or being relied on in cbt master then we would need to make certain we did something like .trim on \ and then re-add the \ if we want to always ensure that enlistmentroot will always end with a \.

Comment (josemesona) [inline build.props]: Technically in the global build.props the trailing backslash is removed with a Trim().  In this context, it is always set to $(MSBuildThisFileDirectory) and will have a trailing backslash so the other properties are assuming it does.  I could have this build.props also trim it if you think it should.

Comment (kingerja) [inline build.props]: for a maintainability aspect having to reason about the context to determine the property value adds additional confusion/risk.  I recommend doing a trim on this property as well so all references to this property will be the same so you don't need to reason if the cbt props were imported or not yet.  This issue I worry about is this build.props others will likely copy as a starting place and their build.props could grow in complexity making the reasoning more than trivial.

Comment (josemesona) [inline build.props]: Okay TrimEnd() added

**Comment by kingerja** [comment]
Change looks good. Go ahead and proceed

**Pull request merged**